Report/Expense/Preview.ts

```diff
@@ -1,18 +1,18 @@
 import { isoly } from "isoly"
-import { Organization } from "@userwidgets/model"
+import { userwidgets } from "@userwidgets/model"
 import { isly } from "isly"
 import { Purchase } from "../../Purchase"
 
 export interface Preview {
 	userExpenses: Record<Purchase["buyer"], Purchase[] | undefined>
-	organization: Organization["id"]
+	organization: userwidgets.Organization["id"]
```

Comment:
userwidgets.Organization.Identifier

Reply:
Was supposed to be the name, but good catch regardless